### PR TITLE
Issue #2311: System spell check is not working with post composer

### PR DESCRIPTION
**Pull request by KevinLeigh**
### Description

Please see description here -> https://github.com/appirio-tech/connect-app/issues/2311

### Changes

- Display spell check errors for post composer

### CODE REVIEW

File: src/components/RichTextArea/RichTextArea.jsx [hunk 1]
```diff
@@ -325,6 +328,7 @@ class RichTextArea extends React.Component {
                     handleKeyCommand={this.handleKeyCommand}
                     plugins={this.plugins}
                     setUploadState={this.setUploadState}
+                    spellCheck
```
Comment (gondzo) [inline src/components/RichTextArea/RichTextArea.jsx]: what is the value of spellCheck variable here?

File: src/components/RichTextArea/RichTextArea.jsx [hunk 2]
```diff
@@ -114,10 +115,11 @@ class RichTextArea extends React.Component {
   }
 
   clearState() {
+    const contentState = EditorState.getCurrentContent()
     this.setState({
       editorExpanded: this.props.editMode,
       titleValue: '',
-      editorState: EditorState.push(this.state.editorState, EditorState.createEmpty().getCurrentContent()),
+      editorState: EditorState.push(this.state.editorState, contentState, EditorState.createEmpty().getCurrentContent()),
```
Comment (gondzo) [inline src/components/RichTextArea/RichTextArea.jsx]: why do we need this change?

Comment (KevinLeigh) [reply src/components/RichTextArea/RichTextArea.jsx]: Yes, `EditorState.push` method takes three parameters which is the `editorState`, `contentState` and `content`. Before there was a missing parameter.

File: src/components/RichTextArea/RichTextArea.jsx (hunk 1 above)
Comment (KevinLeigh) [reply src/components/RichTextArea/RichTextArea.jsx]: The value is equal to `spellCheck={true}`, if I use `spellCheck={true}` then `lintjs` throws an error. When running `lint:fix` to resolve error, then `lint` will make code look like that.

**Comment by KevinLeigh** [comment]
Hey @gondzo, I apologise for the late reply, please see comments above.

**Comment by KevinLeigh** [comment]
@gondzo Hey there, any update on the above comments ?

File: src/components/RichTextArea/RichTextArea.jsx (hunk 2 above)
Comment (gondzo) [reply src/components/RichTextArea/RichTextArea.jsx]: the last parameter is change type, not content https://github.com/facebook/draft-js/blob/196bdab58166395efbb6bbc1f93861907827a7c8/src/model/immutable/EditorState.js#L341

**Comment by KevinLeigh** [comment]
@gondzo Hey there, I reverted the changes you commented on

**Pull request merged**